modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -544,6 +544,7 @@ private function getLangKey(){
 		global $coreLanguageNames;
 		$playerConfig = $this->getUiConfResult()->getPlayerConfig();
 		if( isset( $playerConfig['vars']['localizationCode'] ) ){
+			$playerConfig['vars']['localizationCode'] =  strlen($playerConfig['vars']['localizationCode']) > 2 ? substr($playerConfig['vars']['localizationCode'], 0, 2) :  $playerConfig['vars']['localizationCode'];
```

Comment:
@MaksymKorniiov extract method to method and wrote it in a more readable way.
Add method annotation to explain what purpose this method stands for